ale_linters/c/clangtidy.vim

```diff
@@ -0,0 +1,107 @@
+" Author: vdeurzen <tim@kompiler.org>, w0rp <devw0rp@gmail.com>,
+" gagbo <gagbobada@gmail.com>, Andrej Radovic <r.andrej@gmail.com>
+" Description: clang-tidy linter for c files
+
+call ale#Set('c_clangtidy_executable', 'clang-tidy')
+" Set this option to check the checks clang-tidy will apply.
+" The number of checks that can be applied to C files is limited in contrast to
+" C++
+"
+" Here is an incomplete list of C-compatible checks for clang-tidy version 6:
+" bugprone-suspicious-memset-usage
```

Comment:
This is an awfully large comment, here and below. Could you replace this with a link to the clang-tidy documentation?